man/man7/hashbang.7

```diff
@@ -378,7 +170,7 @@ client-server architecture.
 svn - (Subversion) A software versioning and revision control system
 maintained by apache and designed as a successor to CVS
 
-mercurial - A distributed revision control system designed for high
```

Comment:
It seemed to me that the manpage tries to write the programs like you invoke them. Mercurial is invoked as `hg` and there is no symlink. If that is not the case, I can add some comment that it is invoked as `hg` :)